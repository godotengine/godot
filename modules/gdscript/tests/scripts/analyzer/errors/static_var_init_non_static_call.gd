@static_unload

func non_static():
	return "non static"

static var static_var = non_static()

func test():
	print("does not run")
