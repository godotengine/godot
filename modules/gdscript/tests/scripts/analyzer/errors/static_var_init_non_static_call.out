GDTEST_ANALYZER_ERROR
Cannot call non-static function "non_static()" from a static variable initializer.
